src/lib.rs

```diff
@@ -147,6 +147,59 @@
 //! See the [`platform`] module for documentation on platform-specific cargo
 //! features.
 //!
+//! # Platform/Architecture Support
```

Comment:
Either `#![doc = include_str!("PLATFORM_SUPPORT.md")]`, or to my preference, just avoid the `PLATFORM_SUPPORT.md` file altogether.